TaskLayer/GlycoSearchTask/GlycoSearchParameters.cs

```diff
@@ -8,22 +8,24 @@ namespace TaskLayer
     public class GlycoSearchParameters
     {
         public GlycoSearchParameters()
-        {        
-            OGlycanDatabasefile = "OGlycan.gdb";
-            NGlycanDatabasefile = "NGlycan.gdb";
-            GlycoSearchType = GlycoSearchType.OGlycanSearch;
-            OxoniumIonFilt = true;
-            DecoyType = DecoyType.Reverse;
-            GlycoSearchTopNum = 50;
-            MaximumOGlycanAllowed = 4;
-        }
+        {
 
```

Comment:
I think the default values need to be kept. Otherwise the starting GUI will be empty.